### DISCUSSION TITLE
Dependabot check fails for package-ecosystem: "swift"

### DISCUSSION THREAD

**Discussion by booiiing** [community/community#70553] [Enterprise] [answered]
### Select Topic Area

Bug

### Body

I am trying to enable dependency graph and dependabot alerts for an iOS app that uses Swift Package Manager and XCodeGen in a private server github enterprise repository.
The `MyProject.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved` is checked in and looks good. The `MyProject.xcodeproj/project.pbxproj` is not checked in, since it is generated by XCodeGen.

The documentation is kind of confusing on this: [this page](https://docs.github.com/en/enterprise-server@3.8/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems) does not list Swift as a supported ecosystem, while [this one](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) does include it.
Same for dependabot: [this page](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem) lists `swift` as a valid YAML value for `package-ecosystem`, but the PR check fails with `The property '#/updates/0/package-ecosystem' value "swift" did not match one of the following values: npm, bundler, composer, maven, mix, cargo, gradle, nuget, gomod, docker, elm, gitsubmodule, github-actions, pip, terraform, pub`

Am I missing something? Do we need to enable additional options in the Enterprise account or repository?
I already tried to `enable-beta-ecosystems: true`, but it didn't change anything.

**Comment by carogalvin** [comment, marked as answer]
@booiiing Do you know if you are using Enterprise Server or Enterprise Cloud account? Swift support was added recently, and Enterprise Server accounts have a lagging release cycle - Swift is supported in Enterprise Server starting at version 3.10.

**Reply by booiiing** [reply]
thanks a lot. we're using enterprise server.

**Reply by riamaria** [reply]
Hi @carogalvin were you able to double check the Swift support for Enterprise Server v3.10?

**Reply by carogalvin** [reply]
@riamaria you can see the supported ecosystems for enterprise server 3.10 here: https://docs.github.com/en/enterprise-server@3.10/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems
